src/app/config.js

```diff
@@ -57,11 +72,22 @@ define(['dojo/has', 'esri/config'], function (has, esriConfig) {
     if (has('agrc-build') === 'prod') {
         // atlas.utah.gov
         window.AGRC.apiKey = 'AGRC-A94B063C533889';
+        window.AGRC.quadWord = 'career-exhibit-panel-stadium';
     } else if (has('agrc-build') === 'stage') {
         // test.mapserv.utah.gov
+        window.AGRC.quadWord = '';
         window.AGRC.apiKey = 'AGRC-AC122FA9671436';
     } else {
         // localhost
+        xhr(require.baseUrl + 'secrets.json', {
+            handleAs: 'json',
+            sync: true
+        }).then(function (secrets) {
+            window.AGRC.quadWord = secrets.quadWord;
+        }, function () {
+            throw 'Error getting secrets!';
+        });
+
         window.AGRC.apiKey = 'AGRC-E5B94F99865799';
```

Comment:
Should this be moved to `secrets.json` as well? Or are we not worried about the web api keys?